sockeye/arguments.py

```diff
@@ -206,18 +208,20 @@ def _parse(value: str):
                 return True
             if value.lower() == "false":
                 return False
-            if "." in value or "e" in value:
+            if value.isdigit():
```

Comment:
I guess we haven't needed nor tested this, but this wouldn't work for negative numbers, right?

Reply:
Good catch!

Reply:
Commit 9f77fb6 adds support for negative integers.